src/app/components/feed/FeedCollaboration.js

```diff
@@ -276,6 +278,18 @@ const FeedCollaboration = ({
           />
         </div>
       )}
+      { !feed.dbid && !permissions['create FeedInvitation'] && (
+        <Alert
+          variant="warning"
+          title={
+            <FormattedMessage
+              id="feedCollaboration.editorWarningCreateFeed"
+              defaultMessage="Contact your workspace admin to add other organizations to this shared feed."
+              description="Warning displayed on feed collaboration box on create feed page when user is an editor and can't invite people to the feed."
+            />
+          }
+        />
+      )}
```

Comment:
Does this mean someone without permission to invite others could be able to create a feed anyway?

Reply:
Yes :shrug:

Reply:
Yes @amoedoamorim we don't want to stop editors from being able to push work forward, but things get a bit more sensitive when it comes to outside contributors, so we want admins to handle that